server/controllers/accesses.js

```diff
@@ -118,7 +118,7 @@ export async function update(req, res) {
             throw new KError('missing password', 400);
 
         await req.preloaded.access.updateAttributes(access);
-        res.sendStatus(200);
+        await fetchAccounts(req, res);
```

Comment:
nit: if you don't send a response, the request will always time out on the client.